quil-rs/src/instruction/mod.rs

```diff
@@ -422,7 +422,11 @@ pub fn get_string_parameter_string(parameters: &[String]) -> String {
         return String::from("");
```

Comment:
Not introduced by these changes, but this should be `String::new()`

Reply:
got the other one, but this one's still here